src/supermarket/.env

```diff
@@ -22,7 +22,6 @@ FIERI_URL=http://localhost:3000/fieri/jobs
 FIERI_KEY=YOUR_FIERI_KEY
 FIERI_FOODCRITIC_FAIL_TAGS=any
 FIERI_FOODCRITIC_TAGS=~FC031 ~FC045
-FIERI_RESULTS_ENDPOINT=http://localhost:3000/api/v1/cookbook-versions
-FIERI_COLLABORATORS_REQUEST=http://localhost:3000/api/v1/cookbooks
+FIERI_SUPERMARKET_ENDPOINT=http://localhost:3000
```

Comment:
After an error from Staging following an upgrade to a nightly version and then some reading on dotenv, I believe that these values in `.env` ought to be defaults appropriate for production (so, in this case, port `13000`). [Development environment overrides](https://github.com/bkeepers/dotenv#multiple-rails-environments) should happen in a dev's `.env.development` or `.env.local`.